datamodel_code_generator/format.py

```diff
@@ -12,17 +14,28 @@
 
 
 def format_code(code: str, python_version: PythonVersion) -> str:
+
+    code = apply_isort(code)
```

Comment:
reversing this order (isort > black rather than black > isort) yields better results when there are imports directly in the (custom) Jinja template

Reply:
OK, I feel good.